jsdrupal/drupal-admin-ui
# Don't render additional widget props as HTML field attributes

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/223

## Screenshot / UI changes

No UI changes.

## Testing instructions
The unknown-prop warning will fire if we attempt to render a DOM element with a prop that is not recognized by React as a legal DOM attribute/property.

This happened on `/node/add/recipe` page where we are sending `entityTypeId` to all of its widget components For example: `booleanCheckbox, FileUploadWidget` etc.

- Navigate to `/node/add/recipe`, what we see now is this `Warning: React does not recognize the `entityTypeId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `entitytypeid` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`.
- After pulling this branch, the error/Warning should go away, as we are assigning the props to a new object and deleting the keys which we do not need.
